### DISCUSSION TITLE
Arch Linux'da  QİTV kurulumunda hata

### DISCUSSION THREAD

**Discussion by selatinabi** [ozankaraali/QiTV#6] [Q&A]
ERROR: Ignored the following versions that require a different python version: 0.1.8 Requires-Python >=3.7,<3.11; 0.1.9 Requires-Python >=3.7,<3.11; 1.0.1 Requires-Python >=3.7,<3.11; 1.0.2 Requires-Python >=3.7,<3.11; 1.0.3 Requires-Python >=3.7,<3.11; 1.1.0 Requires-Python >=3.7,<3.11; 1.1.1 Requires-Python >=3.7,<3.12; 1.2.0 Requires-Python >=3.7,<3.12; 1.2.1 Requires-Python >=3.7,<3.12; 2.0.0 Requires-Python >=3.7,<3.12; 2.1.0 Requires-Python >=3.7,<3.12
ERROR: Could not find a version that satisfies the requirement pyqtdarktheme==2.1.0 (from versions: 0.1.0, 0.1.1, 0.1.2, 0.1.3, 0.1.4, 0.1.5, 0.1.6, 0.1.7)
ERROR: No matching distribution found for pyqtdarktheme==2.1.0

Not:  pip install pyqtdarktheme ile kurdum ama yine hata alıyorum.

**Comment by ozankaraali** [comment]
python sürümünüzü düşürün ya da darktheme’i komple iptal edebilirsiniz.

**Comment by selatinabi** [comment]
Merhaba Ozan
hızlı cevap için teşekkürler
darktheme'i nasıl iptal edebilirim

**Reply by ozankaraali** [reply]
main.py line 3 ve line 32 comment-out yeterli olur
